assets/src/edit-story/utils/getInUseFonts.js

```diff
@@ -51,7 +51,7 @@ export function getInUseFontsForPages(pages) {
 export function getTextSetsForFonts({ fonts, textSets }) {
   return textSets
     .map((currentTextSet) => {
-      const hasFontInUse = currentTextSet.reduce(
+      const hasFontInUse = currentTextSet.elements.reduce(
```

Comment:
Looks like the shape of text sets now looks like this:

![image](https://user-images.githubusercontent.com/22185279/117192405-eb5b4f80-ad9e-11eb-8a6e-44f8010f4892.png)

`textSets` is no longer an array of 'arrays of elements' but rather an array of objects. Each object has the elements on the `elements` property.